account_multicurrency_revaluation/report/currency_unrealized_report.py

```diff
@@ -59,11 +70,32 @@ def get_lines(self):
     def compute_totals(self):
         """Compute the sum of values in self.ordered_lines"""
         totals = dict.fromkeys(self.keys_to_sum, 0.0)
+        checks = dict.fromkeys(self.keys_to_check)
```

Comment:
pls do not use `dict` constructor use `{}.fromkeys`

Reply:
that was to coherent with the line above